### PR TITLE
HPCC-13282 Fix grep -v matching error in createIPListFileExcludeDIP

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner Michael.Gardner@lexisnexis.com

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@richardkchapman bugfix to hpcc-run.sh

**Comment by richardkchapman** [comment]
@xwang2713 Please review

File: initfiles/sbin/hpcc-run.sh.in [hunk 1]
```diff
@@ -61,8 +61,15 @@ createIPListFile(){
...
 
 createIPListFileExcludeDIP(){
-  _file=$1
-  echo "$IPS" | grep -v $DIP  > $_file
+  _input=$1
+  _file=$2
+  rm -f $_file
+  touch $_file
+  while read addr; do
+    if [[ "$DIP" != "$addr" ]]; then
```
Comment (richardkchapman) [inline initfiles/sbin/hpcc-run.sh.in]: Is there any guarantee that $DIP contains a single IP ? I suspect not...

**Comment by richardkchapman** [comment]
I don't think this should be taken for 5.2.0 at this late stage. Please check for other issues in this file and retarget to 5.2.2 or 5.4.0 (depending on the extent of the change)

**Comment by AttilaVamos** [comment]
Automated Smoketest Build success

**Pull request closed without merging**